src/flamegraph/mod.rs

```diff
@@ -41,6 +41,22 @@ pub struct Options {
     pub func_frameattrs: FuncFrameAttrsMap,
     pub direction: Direction,
     pub title: String,
+    pub negate_differentials: bool,
+}
+
+impl Default for Options {
+    fn default() -> Self {
+        Options {
+            colors: Default::default(),
+            bgcolors: Default::default(),
+            hash: Default::default(),
+            consistent_palette: Default::default(),
+            func_frameattrs: Default::default(),
+            direction: Default::default(),
+            title: "Flame Graph".to_string(),
+            negate_differentials: Default::default(),
```

Comment:
I think you can just use:
```rust
Options {
    title: "Flame Graph".to_string(),
    ..Default::default(),
}
```

Reply:
I actually did it that way in the tests. I have no idea why I didn't do that here.

Reply:
OK, now I know why I did that. Using Default::default() here would cause an infinite recursion.

Reply:
Ohh, yeah, I guess that makes sense. Too bad..

Maybe move the ones you are actually setting a value for further up, and all the `Default::default()`s to the bottom?